deploy_tools/fabfile.py

```diff
@@ -64,11 +66,15 @@ def _update_secrets(source_folder):
     run(f'echo "TWITTER_SECRET = \'{current_twitter_secret}\'" > {secrets_file}')
 
 def _update_static_files(source_folder):
+    virtualenv_folder = source_folder + '/../virtualenv'
+    run(f'cd {source_folder} && {virtualenv_folder}/bin/python manage.py bower install')
     run(f'cd {source_folder} && ../virtualenv/bin/python manage.py collectstatic --noinput')
 
 def _update_database(source_folder):
     run(f'cd {source_folder} && ../virtualenv/bin/python manage.py migrate --noinput')
 
 def _update_images(source_folder):
-    run(f'cd {source_folder} && mkdir public_officials/static/public_officials/images/profiles')
+    images_folder = source_folder + '/pubic_officials/static/public_officials/images/profiles'
+    if not exists(images_folder):
+        run(f'cd {source_folder} && mkdir public_officials/static/public_officials/images/profiles')
```

Comment:
@meyerhoferc thank you for being so on top of what has to be done for deployment!